IDR/notebook/scripts/nbconverted/1.describe_screens.py

```diff
@@ -59,73 +58,105 @@ def extract_study_info(session, screen_id):
     return details_
 
 
-def describe_screen(screen_id):
+def clean_channel_item(channel_item):
+    if bool(channel_item[channel_item.find("(") + 1 : channel_item.rfind(")")]):
+        channel_item = re.sub(r"\([^()]*\)", "", channel_item)
+
+    return channel_item.strip()
+
+
+def describe_screen(screen_id, session, sample, imaging_method, study_name):
     """Pull additional metadata info per plate, given screen id
 
     Parameters
     ----------
     screen_id: int
         ID of the screen data set
+    sample: str
```

Comment:
Please add `session` as a parameter here as well.

Reply:
Oh I did this 2 days ago. Not sure why it didn't show up for you...

Added in 2776463a3a43ba4d44f1dc2436ed7cd157865f40.